3rdParty/EndianPortable/include/EndianPortable.h

```diff
@@ -207,6 +207,10 @@
 #define htole32(x) le32toh(x)
 #endif
 
+#ifndef _WIN32
+#include <arpa/inet.h> // for htonl()
+#endif
```

Comment:
I'm not sure why this change is needed, where do we use `htonl` in `computeChecksum()`?

Reply:
We use htons() i Added the comment because we use htonl() in htonll()

Reply:
I'm not sure I understand... where in `computeChecksum()` do we use `htonl` or htonll`? In other words - what will fail if we remove this change?

Reply:
i have switched htons to htobe16